lib/qfp.tsx

```diff
@@ -3,7 +3,7 @@ import { SmdChipLead } from "./SmdChipLead";
 
 export const QFP = ({
   pinCount,
-  fullWidth = 10,
+  fullWidth = pinCount === 80 ? 14 : 10,
```

Comment:
hmm this is a hack, you should derive the width automatically, we shouldn't hard code width values